lamindb_setup/_init_instance.py

```diff
@@ -95,11 +95,16 @@ def register_user(usettings):
         pass
 
 
-def register_user_and_storage_in_instance(isettings: InstanceSettings, usettings):
-    """Register user & storage in DB."""
+def register_initial_objects(isettings: InstanceSettings, usettings):
```

Comment:
Maybe call this register_initial_records()? We never use "objects" for records.

Reply:
done